README.md

```diff
@@ -1 +1,62 @@
-# Portfolio-Project
\ No newline at end of file
+# Portfolio Project
+
+In this project, the header and headline of the mobile view on my portfolio was worked on
+
+# Project Name
+
+**My Portfolio Project**
+
+# Screenshot
+
+## Header Image
+
+![Image of the header](./img/header.png)
+
+## Headline Image
+
+![Image of the headline](./img/headline.png)
+
+
+# Live Website
+
+(https://dammyshittu.github.io/Portfolio-Project/)
```

Comment:
- [ ] Please try to follow template guidelines to make the `live link`.  we can use `[Live Demo Link](https://dammyshittu.github.io/Portfolio-Project/)` to make the readme file more presentable.